index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <nav>
+    <div class="nav-bar">
+      <h1 class="logo">Kajal</h1>
+      <img src="./img/hambug.png" alt="hamburger" class="hambug">
+    </div>
+  </nav>
+
+  <header>
+
+    <div class="container">
+
+      <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
```

Comment:
- [ ] this is the logo should be an `<a>` tag in order to be clickable so please I encourage you to change it to <a> tag and implement the same style on it as in the Figma design.

Reply:
sorry, but this is not logo. this is h1 text of header with my name. please check it😥